docs/migrationguide/14_0_0.md

```diff
@@ -22,9 +22,10 @@
   * Locale `firstDay` renamed `firstDayOfWeek` to match other Prime libraries
 
 ## FileUpload
-### mode='simple'
-  * filename won't be displayed per default when auto='true'. Please set displayFilename='true' if you need it.
-  * validation won't be displayed right after the button anymore. The validation has been reimplemented with PrimeFaces CSV and support for `p:message(s)` and `p:growl`.
+  * `invalidSizeMessage`, `invalidFileMessage`, `fileLimitMessage` are deprecated and will be removed in 15.0.0. PF i18n will be used for it.
+  * `maxFileSize`, `fileLimit`, `allowedTypes` are deprecated and will be removed in 15.0.0. Please attach `p:validateFile` to the `p:fileUpload` component.
+  * Simple FileUpload: Client side validation has been reimplemented with PrimeFaces CSV and support for `p:message(s)`/`p:growl`. Please make sure to enable CSV to use it.
+  * Simple FileUpload: filename won't be displayed per default when `auto`=`true`. Please set `displayFilename`=`true` if you need it.
```

Comment:
filename is not displayed by default when `auto`=`true`, set `displayFilename`=`true` instead if you need it.